packages/date/src/index.js

```diff
@@ -19,6 +19,9 @@ import originalLocale from 'date-fns/locale/en-US/index';
 import buildLocalizeFn from 'date-fns/locale/_lib/buildLocalizeFn';
 import buildFormatLongFn from 'date-fns/locale/_lib/buildFormatLongFn';
 
+require( 'moment-timezone/moment-timezone' );
+require( 'moment-timezone/moment-timezone-utils' );
```

Comment:
Why do we need this require calls here?

Reply:
The limitation with the approach I tried is that it depends on webpack recognizing `moment-timezone` as a dependency, which only happens if it is imported. The reason I used `require` is that my local environment didn't complain about it not being on the package's dependencies, but it could also be an `import`. I'm going to add it as a dependency next to avoid this confusion.

I'm not aware if there's currently a way for webpack to load `moment-timezone` from `unpkg.com` without explicitly importing it; I'd really appreciate it if you have any insights on this.

Reply:
Maybe, we need to create a new webpack entry point for `moment` that combines `moment` and those two require statements 🤔